Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+tenacity = "*"
+boto = "*"
```

Comment:
I do not think you need `boto` (which I think is the first version of `boto3`

Reply:
didn't think this was added..oupzy